lib/Common/Core/ConfigParser.cpp

```diff
@@ -337,19 +337,70 @@ void ConfigParser::ParseConfig(HANDLE hmod, CmdLineArgsParser &parser)
         {
             return;
         }
+
         if (_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")) != 0 || configFile == nullptr)
         {
             return;
         }
     }
+#else
+    // Two-pathed for a couple reasons
+    // 1. PAL doesn't like the ccs option passed in.
+    // 2. _wfullpath is not implemented in the PAL.
+    // Instead, on xplat, we'll check the HOME directory to see if there is
```

Comment:
no particular reason other than it's the closest equivalent on xplat to the desktop on Windows and allows you to keep a jscript.config outside the repo in case you clean the repo